pkgs/development/python-modules/aioguardian/default.nix

```diff
@@ -27,7 +27,9 @@ buildPythonPackage rec {
 
   format = "pyproject";
 
-  nativeBuildInputs = [ poetry ];
+  nativeBuildInputs = [
+    poetry
```

Comment:
Not poetry-core?

Reply:
Not at the moment. The build backend was changed already but the list of requirements is not up-to-date.

https://github.com/bachya/aioguardian/pull/67